packages/graphql/package.json

```diff
@@ -32,5 +47,20 @@
     "lint": "eslint src/ test/ --report-unused-disable-directives --max-warnings=0",
     "lint:fix": "eslint . --report-unused-disable-directives --fix"
   },
-  "private": true
+  "files": [
```

Comment:
Why is this necessary?

Reply:
It's used when installing the package as a dependency: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files